packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,63 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw } from 'web3-utils';
+import { PrivateKeyError } from './errors';
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get address from private key
+ */
+export const privateKeyToAccount = (
+	privateKey: string | Uint8Array,
+): {
+	address: string;
+	privateKey: string;
+	signTransaction: () => boolean; // From 1.x
+	sign: () => boolean;
+	encrypt: () => boolean;
+} => {
+	const stringPrivateKey =
```

Comment:
First validate input privateKey, before processing input: 

- length . (for hex: 64 random hex characters or 32 random bytes)
- null or undefined checks